src/routes/Router.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Navigate, createHashRouter } from 'react-router-dom'
+import ErrorComponent from '../components/error/ErrorComponent'
+import Edit from '../components/pages/Edit'
+import NamespacesPage from '../components/pages/Namespaces'
+import EditorPanel from '../components/panels/EditorPanel'
+import EmptyEditorPanel from '../components/panels/EmptyEditorPanel'
+import PageLayout from './PageLayout'
+
+export const hashRouter = createHashRouter([
```

Comment:
This export doesn't match the name of the file.  Typically only components or classes should be capitalized, so I think this one should be either `router` or `hashRouter` and the file name should match the export